azure-pipelines.perf-test.yml

```diff
@@ -12,18 +12,41 @@ steps:
   # No need for checkout since we're using build artifacts.
   - checkout: none
 
+  # TODO: why is this different from main pipeline? is this really getting the correct artifacts?
+  # I think this could be grabbing the wrong (most recent PR) artifacts rather than the one it's associated with
+  # This run is for PR 20200219.5 build #53232 but is grabbing artifacts from 2020219.6 build #53235
+  # https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=53238&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=559759ef-66f4-5335-a959-132505aae30a
+  # Last PR Build: https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=53232&view=logs&jobId=92058b04-f16a-5035-546c-cae3ad5e2f5f&j=ca395085-040a-526b-2ce8-bdc85f692774&t=25a453a0-e716-5e6e-0bee-a24e8e541880
```

Comment:
@kenotron Are you sure this pipeline is always using the correct artifacts? I feel like it would when PR activity is low, but if you check my comments and links above I feel like it could be using the "most recent" pipeline artifacts rather than the artifacts associate with its PR.